client/components/meta-inputs/meta-select/Edit.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <v-select
+  <component
+    :is="meta.tagging ? 'v-combobox' : 'v-select'"
```

Comment:
Stylewise chips are tag like so tagging as a configuration prop is a bit confusing to me 🤷‍♂️, consider some other option name or a separate (combobox) component 🤔 Make sure it is documented [here](https://github.com/ExtensionEngine/tailor/wiki/Configurable-inputs) afterwards.